### PR TITLE
Customizable URI ID used for offline signature

### PR DESCRIPTION
**Pull request by kober32**
Resolves issue #64

### CODE REVIEW

File: docs/Using-Operations-Service.md [hunk 1]
```diff
@@ -250,6 +250,10 @@ fun onQROperationScanned(scannedCode: String): QROperation {
 An offline operation needs to be __always__ approved with __2-factor scheme__ (password or biometry).
 <!-- end -->
 
+<!-- begin box info -->
+Each offline operation created on the server has an __URI ID__ to define its purpose and configuration. The default value used here is `/operation/authorize/offline` and can be modified with the `uriId` parameter in the `authrorize` method.
```
Comment (TomasKypta) [inline docs/Using-Operations-Service.md:254]: Typo in `authorize`

File: library/src/androidTest/java/IntegrationUtils.kt [hunk 2]
```diff
@@ -95,7 +95,12 @@ class IntegrationUtils {
             pa.commitActivationWithPassword(context, pin)
 
             // COMMIT ACTIVATION ON THE SERVER
-            makeCall<CommitObject>(body, "$cloudServerUrl/registration/commit")
+            val bodyCommit = """
+                {
+                  "externalUserId": "test"
+                }
+                """
```
Comment (TomasKypta) [inline library/src/androidTest/java/IntegrationUtils.kt:102]: This keeps the whitespaces. Shouldn't there be something like  `.trimIndent()` at the end?

**Review by TomasKypta — APPROVED** [review]
Looks OK

File: docs/Using-Operations-Service.md (hunk 1 above)
Comment (kober32) [reply docs/Using-Operations-Service.md]: 👍🏻  good catch

File: library/src/androidTest/java/IntegrationUtils.kt (hunk 2 above)
Comment (kober32) [reply library/src/androidTest/java/IntegrationUtils.kt]: Yop, added.

**Pull request merged**